docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.10.1
+LABEL maintainer "chevdor@gmail.com"
+
+RUN apt-get update && \
+	apt-get upgrade -y && \
+	apt-get install -y cmake pkg-config libssl-dev && \
+	curl https://sh.rustup.rs -sSf | sh && \
+	rustup update nightly && \
+	rustup target add wasm32-unknown-unknown --toolchain nightly && \
+	rustup update stable && \
+	cargo install --git https://github.com/alexcrichton/wasm-gc && \
+	git clone https://github.com/paritytech/polkadot.git && \
```

Comment:
i get error `/bin/sh: 1: git: not found` because you need to install git before you can use it, by adding the following beforehand:
```
apt-get install -y git && \
```